spyder/plugins/variableexplorer/widgets/preferences.py

```diff
@@ -0,0 +1,180 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Spyder Project Contributors
+#
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+# -----------------------------------------------------------------------------
+
+"""
+Dialog window for setting the viewing preferences of the data frame editor.
+"""
+
+# Standard library import
+from typing import Optional
+
+# Third party imports
+from qtpy.QtWidgets import (
+    QDialog, QDialogButtonBox, QGroupBox, QLabel, QLineEdit, QRadioButton,
+    QVBoxLayout, QWidget)
+
+# Local imports
+from spyder.config.base import _
```

Comment:
```suggestion
from spyder.api.translations import _
from spyder.api.widgets.dialogs import SpyderDialogButtonBox
```

- Better to use the API import for `_`.
- The second import is needed below.